src/components/search/index.js

```diff
@@ -16,8 +17,13 @@ import { Root } from "./styles"
 
 import "./styles.css"
 
-const PageHit = () => ({ hit }) => (
-  <div className="text-gray-700 hit">
+const PageHit = () => ({ hit, selected, onMouseHover }) => (
+  <div
+    className={classnames("text-gray-700 hit", {
+      selected: selected === true,
```

Comment:
I know this is convention, but when we are treating a var a boolean, it's fine in my book to not coerce it :)

```
className={classnames("text-gray-700 hit", { selected } )} 
```